server.js

```diff
@@ -26,6 +26,7 @@ io.on('connection', socket => {
   socket.color = Math.floor(Math.random() * 255); // eslint-disable-line no-param-reassign
   console.log('a user connected');
   io.to(`${socket.id}`).emit('message history', messages);
+  io.emit('update roomlist', rooms);
```

Comment:
Går att skriva `io.to('${socket.io}').emit('update roomlist', rooms);` här för att bara uppdatera rumlistan för den som ansluter sig, för de som redan är anslutna har ju redan rätt lista